docs/hour-of-code/educators-2021.html

```diff
@@ -398,12 +398,93 @@ <h5 class="display-font">Assign in Teams</h5>
+                  your classroom! Then throughout CSEW, tune in with your students to
+                  live lessons that provide tips and tricks on how to create a game
+                  MakeCode Arcade game. 
+                </p>
+                <ul class="accent-list">
+                    <li>
+                        Save the Forest | Makecode Arcade HOC Educator Training 
+                         <b class="sunflower">(11/17 or 12/1 8 am & 5 pm PST)</b> 
+                    </li>
+                    <li>
+                        Code with MakeCode Arcade Part 1: The Basics  
+                         <b class="sunflower">(12/7 10:30 am PST)</b> 
+                    </li>
+                    <li>
+                        Code with MakeCode Arcade Part 2: Designing Your Game with Drew Okenfuss 
+                         <b class="sunflower">(12/8 10:30 am PST)</b> 
+                    </li>
+                    <li>
+                        Code with MakeCode Arcade Part 3: Developing your Game with MakeCode Engineers 
+                         <b class="sunflower">(12/10 10:30 am PST)</b> 
+                    </li>
+                    <li>
+                        Quick Start: Creating Your Game Jam Project  
+                         <b class="sunflower">(12/10)</b> 
+                    </li>
+                </ul>
+                <a class="btn" href="https://aka.ms/CSEW2021" target="_blank" rel="noreferrer"
+                        ><h5 class="display-font">Sign Up</h5></a
+                    >
+            </section>
             <section class="hoc-section">
                 <h3 class="display-font title sunflower">
                     Resources for Further Exploration
                 </h3>
                 <ul class="accent-list">
+                    <li>
+                        Game Jam
+                        <ul class="accent-list">
+                            <li>
+                                <a
+                                    href="https://arcade.makecode.com/gamejam/global-2021"
+                                    target="_blank"
+                                    rel="noreferrer"
+                                    >Global MakeCode Arcde Game Jam</a
+                                >
+                            </li>
+                            <li>
+                                <a
+                                    href="https://arcade.makecode.com/gamejam/lessons/box"
+                                    target="_blank"
+                                    rel="noreferrer"
+                                    >Game Jam Guide</a
+                                >
+                            </li>
+                            <li>
+                                <a
+                                    href="https://microsoft.sharepoint.com/:w:/t/Prime8xMakeCode/ER2V7ZEkYQNAqcMRdCcPmCsBY-b_lWsJv-NFPratwlhIPA?e=DyEp4S"
```

Comment:
This points to a vendor SharePoint site?  Is this accessible externally?  Anyway, I would remove this link and the next two - these are materials that are part of the Game Jam in a Box, which we already referenced above.